apis/infrastructure/v1beta1/byocluster_webhook.go

```diff
@@ -63,10 +61,5 @@ func (byoCluster *ByoCluster) ValidateDelete() error {
 }
 
 func validateByoClusterSpec(spec ByoClusterSpec) field.ErrorList {
```

Comment:
This webhook is pretty much non-functional and empty. Do we even need this anymore?

Reply:
It is affecting a lot of code which required all the more refactoring, that's why didn't touch it, but it is doable.

Reply:
Seems like we'll be removing `byoclustertemplate_webhook` as part of #573. We should similarly completely remove the `byocluster_webhook` here. WDYT?

Reply:
That makes sense to me, removed `byocluster_webhook`. Thanks for the suggestion.